OHHTTPStubs/UnitTests/Test Suites/MocktailTests.m

```diff
@@ -110,18 +111,18 @@ - (NSHTTPURLResponse *)runLogin
     NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:url
                                                            cachePolicy:NSURLRequestUseProtocolCachePolicy
                                                        timeoutInterval:60.0];
-    
+
```

Comment:
@AliSoftware, Sorry I missed that. It's fixed now. My Xcode is already up as you mentioned, but I just didn't CTRL-i to repair the indentation on that one file. It's in the new commit of this PR now.